qiskit/circuit/library/evolution/evolution_synthesis.py

```diff
@@ -0,0 +1,44 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Evolution synthesis."""
+
+from typing import List, Union
+from abc import ABC, abstractmethod
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+from qiskit.quantum_info import SparsePauliOp
+
+
+class EvolutionSynthesis(ABC):
+    """Interface for evolution synthesis algorithms.
+
+    TODO do we need this or do we prefer plain functions?
+    """
+
+    @abstractmethod
+    def synthesize(
+        self, operators: Union[SparsePauliOp, List[SparsePauliOp]], time: float
+    ) -> QuantumCircuit:
+        """Synthesize the evolution of a (list of) operator(s) to a circuit.
+
+        If a single operator, the summands are assumed not to commute.
+        If a list, the lust elements are *not* assumed to commute pairwisely, however summands in
```

Comment:
```suggestion
        If a list, the list elements are *not* assumed to commute pairwisely, however summands in
```
I imagine this is what was meant - though `its elements are...` would work too